### PR TITLE
Added metadata for block.py file

### PR DESCRIPTION
**Pull request by Med16-11**
- [x] issues: fixes #11765
- [ ] tests added / passed
- [ ] release document entry (if new feature or API change)

### CODE REVIEW

File: examples/basic/areas/block.py [hunk 1]
```diff
@@ -1,3 +1,13 @@
+"""
+Creates a block plot using Bokeh and displays it using the show() function.
+
+.. bokeh-example-metadata::
+    :sampledata: None
+    :apis: bokeh.plotting.figure, bokeh.plotting.show, bokeh.models.Block
+    :refs: :ref:`uuserguide_plotting_visualization_glyphs`
```
Comment (bryevdv) [inline examples/basic/areas/block.py]: The references are a little obscure, they need to match the ones that exist in the docs source files, in this case:
```suggestion
    :refs: :ref:`ug_basic_areas_rects`
```

File: examples/basic/areas/block.py [hunk 2]
```diff
@@ -1,3 +1,13 @@
+"""
+Creates a block plot using Bokeh and displays it using the show() function.
```
Comment (bryevdv) [inline examples/basic/areas/block.py]: nit: style consistency with existing docstrings
```suggestion
""" Creates a block plot using Bokeh and displays it using the show() function.
```

**Comment by bryevdv** [comment]
A few very small comments, @tcmetzger can also take a look over towards the end of this week

**Comment by Med16-11** [comment]
Thanks for your suggestions @bryevdv ! I have made all the suggested changes.

**Review by tcmetzger — APPROVED** [review]
Looks great, thank you!

**Pull request merged**